src/test/java/com/micetr0/controller/NoteControllerTest.java

```diff
@@ -0,0 +1,12 @@
+package com.micetr0.controller;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class NoteControllerTest {
+
+    @Test
+    public void addNote() {
```

Comment:
need test cases

Reply:
Note controller still has to be implemented. Will be addressed in future pull request